lib/api-sim.js

```diff
@@ -7,7 +7,11 @@ let cls;
 module.exports = function (appoptics) {
   ao = appoptics;
   aob = ao.addon;
-  cls = ao.cls;
+  // check for property to avoid triggering node 14 warning on
+  // non-existent property in unfinished module.exports
+  if (ao.hasOwnProperty('cls')) {
```

Comment:
a suggestion:
`if ('cls' in ao) {` seems to be the pattern used in this code base for testing properties
`hasOwnProperty` is a bit more restrictive, as it would return false for inherited properties
Less restrictive may be more future-proof

Reply:
I don't have any insight into which approach is preferable, but can confirm that the `'cls' in ao` style also prevents the warning from appearing.

Reply:
Good to know :). I did consider the `property in object` style checking but decided against it since these two properties are very specific to the agent and I didn't see a scenario where the agent would try inheriting them from some parent object.